Description: Remove usage of bundler
 Prevent using bundler to install needed gems.
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2015-07-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -1,5 +1,4 @@
 require 'rubygems'
-require 'bundler/setup'
 require 'seed-fu'
 require 'logger'
 
